sklearn/svm/classes.py

```diff
@@ -1012,6 +1012,13 @@ class OneClassSVM(BaseLibSVM):
     intercept_ : array, shape = [n_classes-1]
         Constants in decision function.
 
+    offset_ : float
+        Offset used to define the decision function from the raw scores.
+        We have the relation: decision_function = score_samples - offset_.
+        The offset is equal to intercept_ and is provided for consistency
+        with other outlier detection algorithms such as LocalOutlierFactor,
+        IsolationForest and EllipticEnvelope.
```

Comment:
I would remove `such as LocalOutlierFactor, IsolationForest and EllipticEnvelope`. Otherwise if a new outlier detection estimator is added to scikit-learn we would need to update the list.